sdk/state/close_test.go

```diff
@@ -27,15 +27,15 @@ func TestChannel_CloseTx(t *testing.T) {
 		LocalEscrowAccount:  localEscrowAccount,
 		RemoteEscrowAccount: remoteEscrowAccount,
 	})
-	channel.openAgreement = OpenAgreement{
+	oa := OpenAgreement{
```

Comment:
I needed to build the txs with `closeTxs` using the agreement before setting the agreement in the channel because the channel looks at the agreement to decide if it should reuse the stored tx.